config/locales/en/course/forum/topics.yml

```diff
@@ -17,3 +17,27 @@ en:
         destroy:
           success: 'Topic %{title} was deleted.'
           failure: 'Failed to delete topic: %{error}'
+        subscribe:
+          tag: 'Subscribe'
+          success: 'You have subscribed to Topic %{title}.'
```

Comment:
But we do not do this for all the other components right? for consistency, just say 'successfully subscribed to topic' should do